entrypoint.sh

```diff
@@ -51,7 +51,15 @@ SETTINGS_PY="/etc/zulip/settings.py"
 # === initialConfiguration ===
 prepareDirectories() {
     mkdir -p "$DATA_DIR" "$DATA_DIR/backups" "$DATA_DIR/certs" "$DATA_DIR/letsencrypt" "$DATA_DIR/uploads"
-    [ -e /etc/letsencrypt ] || ln -ns "$DATA_DIR/letsencrypt" /etc/letsencrypt
+
+	# See commentary in the Dockerfile about this process.
```

Comment:
Nit: the surrounding indentation is space-based; don’t mix it with tabs.

Reply:
Oops: thanks. My tabstop is set to the same as the space indents here and I guess auto-detect didn't work. Amending...